### PR TITLE
Feature/pp 60370

### PR DESCRIPTION
**Pull request by ashraf-km**
### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing](../../paypayopa-sdk-php/blob/master/contributing.md) document?
* [ ] Have you read and signed the automated Contributor's License Agreement?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../paypayopa-sdk-php/pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,4 +24,8 @@ run_coverage:
 	vendor/bin/phpunit --coverage-clover build/logs/clover.xml --testdox --debug -c phpunit.xml.dist 
 	vendor/bin/php-coveralls -v
 coverall_upload:
-	vendor/bin/php-coveralls --coverage_clover=build/logs/clover.xml -v
\ No newline at end of file
+	vendor/bin/php-coveralls --coverage_clover=build/logs/clover.xml -v
+run_mock1:
```
Comment (shreyanshp) [inline Makefile]: @solancer Why is this needed here?

**Review by solancer — APPROVED** [review]

File: src/Controllers/CashBack.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+     *
+     * @param CashBackPayload $payload SDK payload object
+     * @return array
+     * @throws ClientControllerException
+     * @throws ModelException
+     */
+    public function giveCashback($payload)
+    {
+        // unset($payload->merchantCashbackReversalId);
+        $payload->merchantCashbackReversalId = "dummy";
```
Comment (shreyanshp) [inline src/Controllers/CashBack.php:40]: why is the default value dummy here instead of just failing?
How is this handled in python?
@solancer

Comment (shreyanshp) [reply src/Controllers/CashBack.php:40]: @solancer Fix / Update this in next PR, please

**Review by shreyanshp — APPROVED** [review]
